[ADD] DNS hoster INWX (#19)
felixfontein/ansible-acme: docs/docsite/rst/general_role_parameters.rst

If DNS challenges are used, the following variables define how the challenges can be fulfilled:

> - ``acme_certificate_dns_provider``: must be one of ``route53``, ``hosttech``, ``ns1`` and ``inwx``. Each needs more information:

Reviewer: ```suggestion
- ``acme_certificate_dns_provider``: must be one of ``route53``, ``hosttech``, ``ns1``, and ``inwx``. Each needs more information:
```
I really like the Oxford comma :)

  - For ``route53`` (`Amazon Route 53 <https://aws.amazon.com/route53/>`_), the credentials must be passed as ``acme_certificate_aws_access_key`` and ``acme_certificate_aws_secret_key``.
  - For ``hosttech`` (`hosttech GmbH <https://www.hosttech.ch/>`_), the credentials have to be passed as ``acme_certificate_hosttech_username`` and ``acme_certificate_hosttech_password`` for using the old WSDL API, and ``acme_certificate_hosttech_token`` for the new JSON API.
  - For ``ns1`` (`ns1.com <https://ns1.com>`_) the key for your API account must be passed as ``acme_certificate_ns1_secret_key``. Also it depends on external module ``ns1_record``. See below for instructions on how to install these modules.
  - For ``inwx`` (`inwx.de <https://inwx.de>`_) the credentials have to be passed as ``acme_certificate_inwx_username`` and ``acme_certificate_inwx_password``. Please keep in mind that 2FA (two factor authentication) is currently not supported and needs do be disabled. Related Issue: (`inwx/ansible-collection#3 <https://github.com/inwx/ansible-collection/issues/3>`_)

> Please note that the DNS challenge code is not perfect. The Route 53, Hosttech, NS1 and INWX functionality has been tested.

Reviewer: ```suggestion
Please note that the DNS challenge code is not perfect. The Route 53, Hosttech, NS1, and INWX functionality has been tested.
```

    curl --create-dirs -L -o ~/.ansible/plugins/modules/ns1_record.py https://github.com/ns1/ns1-ansible-modules/raw/master/library/ns1_record.py

Once NS1 converts their `set of modules <https://github.com/ns1/ns1-ansible-modules>`_ into a `collection <https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html>`_, it will become a lot easier to install and use them (`NS1 tracking issue <https://github.com/ns1/ns1-ansible-modules/issues/32>`_).

Setting up INWX modules

> ~~~~~~~~~~~~~~~~~~~~~~

Reviewer: ```suggestion
~~~~~~~~~~~~~~~~~~~~~~~
```

For ``inwx`` (`inwx.de <https://inwx.de>`_) the available Ansible Galaxy collection ``inwx.collection`` (`galaxy.ansible.com <https://galaxy.ansible.com/inwx/collection>`_) needs to be installed.
  

> It's as simple as: ``ansible-galaxy collection install inwx.collection``

Reviewer: ```suggestion
It is as simple as: ``ansible-galaxy collection install inwx.collection``
```
